PIE-604 PLAY new copy

dimarconicola opened the pull request:
##### Description
<!-- A description on what this PR aims to solve. -->

##### Checklist
<!-- Remove items that do not apply. For completed items, change [ ] to [x]. -->

- [ ] Linter status: 100% pass
- [ ] Changes don't break existing behavior
- [ ] Test coverage hasn't decreased

##### Testing
<!-- Why should the PR reviewer trust that this change doesn't break
  anything? How have you tested this change?
-->

##### Refers/Fixes
<!--
  Link to an issue if applicable. For example:
  If your PR fixes an issue -- Fixes: #102
  If your PR refers an issue -- Refs: #101
-->

dimarconicola committed:
PLAY new copy

dimarconicola committed:
typo

dimarconicola committed:
typos

AdrianAddesso committed:
removed incorrect underlinee

removed underline from 
, covering:

AdrianAddesso reviewed (approved):
I approve the comments

AdrianAddesso committed:
Copy change

1. Minimum allocation of 2% for each underlying asset. Introduced to ensure minimum slippage when minting at scale 
TO BE
1. Minimum allocation of 2% for each underlying asset.

AdrianAddesso committed:
removed a period

Minimum allocation of 2% for each underlying asset.
to
Minimum allocation of 2% for each underlying asset
